### PR TITLE
Add (script/*): create several new script

### LINKED ISSUE #109: New script: etcher
https://www.balena.io/etcher/

### LINKED ISSUE #110: New script: ramme
https://github.com/terkelg/ramme/tree/v3.2.5

### LINKED ISSUE #111: New script: windows 95
https://github.com/felixrieseberg/windows95/tree/v2.0.0

### LINKED ISSUE #112: New script: tweet-tray
https://github.com/jonathontoon/tweet-tray#downloads

### LINKED ISSUE #113: New script: simplenote
https://github.com/Automattic/simplenote-electron/releases

### LINKED ISSUE #114: New script: Notable
https://github.com/notable/notable/releases/tag/v1.3.0

### LINKED ISSUE #115: New script: rofi last version
https://github.com/DaveDavenport/rofi/blob/next/INSTALL.md

### PR DESCRIPTION
**Pull request by Wabri**
new file:   etcher.sh
	new file:   franz.sh
	new file:   notable.sh
	new file:   ramme.sh
	new file:   rofi.sh
	new file:   simplenote.sh
	new file:   telegram.sh
	new file:   tweet-tray.sh
	new file:   windows95.sh

Closes #109
Closes #110
Closes #111
Closes #112
Closes #113
Closes #114

### CODE REVIEW

File: scripts/etcher.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Ramme unofficial Instagram Desktop App - https://github.com/balena-io/etcher
```
Comment (IanDuncanT) [inline scripts/etcher.sh]: This description looks wrong for this file.

File: scripts/README.md [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
 - Nvidiadriver - NVIDIA driver support of Geforce 4xx and higher GPUs - https://wiki.debian.org/NvidiaGraphicsDrivers
 - Light - Light GNU/Linux application to control backlights - http://haikarainen.github.io/light
+- Tweet-tray - Tweet-tray tweet quickly from the desktop without any distractions - https://github.com/jonathontoon/tweet-tray
 - Oh-my-zsh - zsh + oh-my-zsh - https://ohmyz.sh/
 - Virtualbox - VirtualBox is a general-purpose full virtualizer for x86 hardware - https://www.virtualbox.org
+- Springtoolsuite - Spring Tools 4 is the next generation of Spring tooling for your favorite coding environment - https://spring.io/tools
 - Steam - Steam ultimate entertainment platform - https://store.steampowered.com/about
 - Pycharm - Pycharm Community edition - https://www.jetbrains.com/pycharm/
 - Dropbox - Dropbox cloud storage - https://www.dropbox.com/
+- Etcher - Ramme unofficial Instagram Desktop App - https://github.com/balena-io/etcher
```
Comment (IanDuncanT) [inline scripts/README.md]: This needs fixed again since readme updater was not run again.

**Comment by Wabri** [comment]
Because i cannot resolve some dependencies

**Comment by Wabri** [comment]
You can see the error here #115

**Pull request merged**